doc/REST-interface.md

```diff
@@ -89,10 +89,12 @@ Only supports JSON as output format.
 * softforks : (array) status of softforks in progress
 
 #### Query UTXO set
-`GET /rest/getutxos/<checkmempool>/<txid>-<n>/<txid>-<n>/.../<txid>-<n>.<bin|hex|json>`
+`GET /rest/getutxos/<TXID>-<N>/<TXID>-<N>/.../<TXID>-<N>.<bin|hex|json>`
+`GET /rest/getutxos/checkmempool/<TXID>-<N>/<TXID>-<N>/.../<TXID>-<N>.<bin|hex|json>`
```

Comment:
I agree with @shaavan on adding a line break here (and for https://github.com/bitcoin/bitcoin/blob/master/doc/REST-interface.md#blocks; may as well sneak it into the commit if you agree).

Reply:
Agree that this is a good opportunity to also fix this for the blocks section. Done.